src/org/parosproxy/paros/core/scanner/VariantAbstractRPCQuery.java

```diff
@@ -17,7 +17,7 @@
  * See the License for the specific language governing permissions and 
  * limitations under the License. 
  */
-
+// ZAP: 2018/01/16 Added the variantType variable and a new constructor that sets this variable.

```

Comment:
Note this is a ZAP class (same for `VariantJSONQuery` and `VariantJSONQueryUnitTest`).

Reply:
Bump, the comment should be removed, ZAP class :)